components/script/fetch.rs

```diff
@@ -205,9 +211,10 @@ pub(crate) fn Fetch(
         signal.Reason(cx, abort_reason.handle_mut());
         abort_fetch_call(
             promise.clone(),
-            &request,
-            None,
+            &request_object,
+            Some(&response),
```

Comment:
This was only for testing to see if it works. It should actually be `None`, but wasn't sure yet. Since many of the tests don't run, will keep this here for now.